OpenSim/Common/ComponentSocket.h

```diff
@@ -960,13 +959,13 @@ class Input : public AbstractInput {
     /** @{                                                               */ \
     /** comment                                                          */ \
     /** In an XML file, you can set this socket's connectee name         */ \
-    /** via the <b>\<socket_##cname##_connectee_name\></b> element.      */ \
+    /** via the <b>\<socket_##cname\></b> element.                       */ \
     /** See AbstractSocket for more information.                         */ \
     /** @see connectsocket_##cname##()                                   */ \
     OpenSim_DOXYGEN_Q_PROPERTY(T, cname)                                    \
     /** @}                                                               */ \
     /** @cond                                                            */ \
-    PropertyIndex PropertyIndex_socket_##cname##_connectee_name {           \
+    PropertyIndex PropertyIndex_socket_##cname {           \
```

Comment:
Can you fix the whitespace here and on 894?

Reply:
Done